pkg/model/application.proto

```diff
@@ -85,12 +85,23 @@ message ApplicationSyncState {
     int64 timestamp = 5 [(validate.rules).int64.gt = 0];
 }
 
+enum ApplicationComponentKind {
+    CONTAINER_IMAGE = 0;
+}
+
+message ApplicationComponentVersion {
+    ApplicationComponentKind kind = 1;
+    stirng version = 2;
+    string url = 3;
+}
+
 message ApplicationDeploymentReference {
     string deployment_id = 1 [(validate.rules).string.min_len = 1];
     DeploymentTrigger trigger = 2 [(validate.rules).message.required = true];
     string summary = 3;
     string version = 4;
-    string config_filename = 5;
```

Comment:
The index number must be kept in order to avoid breaking change.
It means that with this change, the old Piped can not talk with the new server which is running with the new data layout.

Reply:
I see. I'll fix it!